app/jobs/send_outbound_call_status_job.rb

```diff
@@ -0,0 +1,23 @@
+class SendOutboundCallStatusJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(user_id, offer_id, status)
+    user  = User.find_by(id: user_id)
+    offer = Offer.find_by(id: offer_id)
+
+    if(user && offer)
+      text = if status == "completed"
+          "Called #{offer.created_by.full_name}"
+        else
+          "Call attempt failed: #{status.titleize}"
+        end
+
```

Comment:
@swatijadhav Think I commented somewhere already but I don't think this needs to be an active job as it is only creating one DB entry. i.e. it isn't calling an external service that has significant latency. Just a service class that handles the details should be fine. e.g. app/services/send_outbound_call_status.rb